pkg/status/status_test.go

```diff
@@ -579,113 +621,12 @@ func TestPartialPodCancellation(t *testing.T) {
 	}
 }
 
-func TestCancelIngressProbing(t *testing.T) {
```

Comment:
Can you elaborate why this is no longer needed?

Reply:
Was dropped unintentionally